apps/dapp-console/app/components/Header/HeaderTab.tsx

```diff
@@ -0,0 +1,50 @@
+import { Route, routes } from '@/app/constants'
+import { cn } from '@/app/lib/utils'
+import Link from 'next/link'
+
+type HeaderTabsProps = {
+  currentRoute: Route['path']
+}
+
+const HeaderTabs = ({ currentRoute }: HeaderTabsProps) => {
+  return (
+    <div className="flex gap-8 items-end h-full">
+      {Object.keys(routes).map((route) => {
```

Comment:
nit: will be better to make list of supported navbar routes an explicit array

```
const headerTabRoutes = [routes.CONSOLE, routes.INSIGHTS]

      {headerTabRoutes.map(({ path, label }) => {
           ....
       }

```

1. since element ordering can be unexpected in javascript objects
2. likely we might not support all possible routes in the navbar, just a subset